src/bindings/python/src/pyopenvino/frontend/frontend.cpp

```diff
@@ -137,7 +137,29 @@ void regclass_frontend_FrontEnd(py::module m) {
             )");
 
     fem.def("add_extension",
```

Comment:
thanks for documenting this, I will try to cover remaining methods in the next PRs